### ISSUE TITLE
Remove typ from Token status data

### ISSUE THREAD

**Issue by paulbastian** [vcstuff/draft-looker-jwt-cwt-status-list#14]
The current proposal for the JWT having a status is:
```
{
  "alg": "ES256",
  "kid": "11"
}
.
{
  "iss": "https://example.com",
  "sts": {
    "typ": "revocation-list",
    "idx": 0,
    "uri": "https://example.com/statuslists/1"
  }
}
```
In my opinion the `typ`claim here is absolete.
The type of the status list should be known from the `typ`claim in the Status List JWT.
How to fetch the Status List JWT is given by the specification or by the scheme in `uri`.

**Comment by paulbastian** [comment]
Also remove the phrase "The value MUST be equal to that of the "typ" attribute in the "sts" claim for the token who's status is being validated" as this is obsolete in this case as well

**Comment by tplooker** [comment]
Im not against this proposed simplification however removing it would prevent an extensibility point based of the 'typ' value to control what types of uri could be found in the URI e.g are these always https? or are other schemes supported?

**Comment by paulbastian** [comment]
I agree with @c2bo . The scheme is probably enough. I would suggest making HTTPS schemes a MUST requirement and other schemes MAY be used

**Comment by paulbastian** [comment]
This is addressed by #17
